delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/resources/ResourceRejection.scala

```diff
@@ -98,6 +98,12 @@ object ResourceRejection {
         s"Resource '$id' failed to validate against the constraints defined in schema '$schema'"
       )
 
+  final case class InvalidSchemaRejection(
```

Comment:
Is there a reason why you decided to not wrap schema errors here, like we do for projects or orgs?

Reply:
The errors we get here are now errors related to resolution via resolvers, not directly schemas.
The project/org related ones are already risen by fetching the project.
The only one that is left is related to schema deprecation and it was weird and verbose to raise a wrapped one in resources state machine to convert it back later.